controllers/storagecluster/cephblockpools.go

```diff
@@ -14,18 +14,16 @@ import (
 
 type ocsCephBlockPools struct{}
 
-func generateCephReplicatedSpec(initData *ocsv1.StorageCluster) cephv1.ReplicatedSpec {
-	if arbiterEnabled(initData) {
-		return cephv1.ReplicatedSpec{
-			Size:                     4,
-			TargetSizeRatio:          .49,
-			ReplicasPerFailureDomain: 2,
-		}
-	}
-	return cephv1.ReplicatedSpec{
-		Size:            3,
-		TargetSizeRatio: .49,
+func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string) cephv1.ReplicatedSpec {
```

Comment:
Since this function is going to be used for multiple CRs, I'd move it to a different file. Not sure if it should be an existing one or a new one, I'll leave that up to you.

Reply:
Moved to generate.go. It is the closest in usage, generates artifacts considering ceph related configuration in the storage cluster spec.

Let me know if you want to move the other newly introduced get functions too. I was thinking of moving them to defaults.go but left them as is.